cmd/juju/commands/helptool_test.go

```diff
@@ -39,6 +39,7 @@ Currently available charm hook tools are:
     close-port               ensure a port or range is always closed
     config-get               print application configuration
     container-spec-set       set container spec information
+    goal-state               print charms desired configuration
```

Comment:
Please change the description. It's not the desired configuration (configuration is a separate concept).
"print the status of the charm's peers and related units"